src/Raven.Client/Documents/Smuggler/DatabaseSmugglerOptions.cs

```diff
@@ -87,4 +87,10 @@ internal interface IDatabaseSmugglerOptions
         int MaxStepsForTransformScript { get; set; }
         List<string> Collections { get; set; }
     }
+
+    public enum ExportCompressionAlgorithm
```

Comment:
Maybe just `CompressionAlgorithm` ? So it will be a more universal name. Neither specific to export or backup